input/fsh/modules/biospecimen.fsh

```diff
@@ -0,0 +1,140 @@
+/*Biospecimen Collection Module*/
+
+Logical: CdmBiospecimenCollection
+Id: SharedDataModelBiospecimenCollection
+Title: "Shared Data Model for Biospecimen Collection"
+Description: "Shared Data Model for Biospecimen Collection"
+* sampleGenerated 1..1 reference "Sample that was generated from this collection event"
+* participant 1..1 reference "The participant from whom the biospecimen was taken"
+* ageAtCondition 0..1 Quantity "The age at which this biospecimen was collected. Could be expressed with a term, an age, or an age range."
+* method 0..1 code "The approach used to collect the biospecimen"
+* site 0..1 code "The location of the specimen collection"
+* spatial 0..1 code "Any spatial/location qualifiers"
+* laterality 0..1 code "Any spatial/location qualifiers"
+
+Extension: BiospecimenSpatial
+Id: biospecimen-spatial
+Title: "Spatial Information"
+Description: "Spatial Information"
+* insert SetContext(Specimen.collection)
```

Comment:
I don't remember where we are about the codes that should be used here. But, I think it's worth letting folks know that we need some meaningful terms for these fields. I don't think it should hold the PR up, but definitely something to remind people about at these meetings.

Reply:
What I did was create these extensions which could use the codes for $body-location-qualifier or $laterality-qualifier in the Alias file but that is also not fully tested in the example since neither of those fields are supplied in the example. I'll put a comment in the script to remind us to keep thinking on these terms!